README.md

```diff
@@ -1 +1,16 @@
-# Hello-Microverse-project
\ No newline at end of file
+📖 [Setup a "Hello Microverse" project]
+This excercise intends to reinforce knowledge about how projects must be developed in order to get a good code review experience in Microverse program.
+
+
+Key Features
+
+[Create branches on GitHub]
+[Adding a .gitignore file]
+[Setting linters in projects]
+[Asking for a code reviewing]
+
+
+👤 Author1
+
+GitHub: @oovillagran
+Twitter: @oovillagran
```

Comment:
- [ ] Nice work making efforts on this section of the README. However, you need to follow the project requirements concerning the [README TEMPLATE](https://github.com/microverseinc/readme-template) given. 
- All the headers on the file should be present with the exception of the FAQ  which is optional

## Expected header:
![image](https://user-images.githubusercontent.com/50990925/218892762-4168fe3c-e93e-488f-8624-141040c8481c.png)